cli/command/command.go

```diff
@@ -69,8 +76,20 @@ func PortCommand(factory app.ProjectFactory) cli.Command {
 // UpCommand defines the libcompose up subcommand.
 func UpCommand(factory app.ProjectFactory) cli.Command {
 	return cli.Command{
-		Name:   "up",
-		Usage:  "Bring all services up",
+		Name:      "up",
+		Usage:     "Builds, (re)creates, starts, and attaches to containers for a service.",
+		ArgsUsage: "[SERVICE...]",
+		Description: "Unless they are already running, this command also starts any linked services.\n\n" +
+			"The `libcompose-cli up` command aggregates the output of each container. When\n" +
```

Comment:
@thaJeztah I wonder, I could also use *backticks* to not have to escape the carriage-return but.. I had trouble with them for [these](https://github.com/docker/libcompose/pull/126/files#diff-3145698b159fb0a026984b4a017a8b24R25) (backticks inside backticks…).